.github/workflows/main.yml

```diff
@@ -0,0 +1,30 @@
+name: Ruby
+
+on:
+  push:
+    branches:
+      - master
+
+  pull_request:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    name: Ruby ${{ matrix.ruby }}
+    strategy:
+      matrix:
+        ruby:
+          - '3.0.6'
+          - '3.1.4'
+          - '3.2.2'
+          - '3.3.0'
```

Comment:
I would make these minor versions. Do we really need to test < 3.2?

Reply:
I figured we might as well since the tests pass.  We could keep testing < 3.2 as long as the ruby versions are supported and the tests pass.  But I'm also happy to remove them if you'd prefer!